src/js/components/MaskedInput/index.d.ts

```diff
@@ -21,9 +21,13 @@ export interface MaskedInputProps {
   value?: string | number;
 }
 
-declare const MaskedInput: React.ComponentClass<MaskedInputProps &
-  Omit<JSX.IntrinsicElements['input'], keyof MaskedInputProps>>;
-export type MaskedInputType = MaskedInputProps &
-  Omit<JSX.IntrinsicElements['input'], keyof MaskedInputProps>;
+export interface MaskedInputExtendedProps
+  extends MaskedInputProps,
+    Omit<JSX.IntrinsicElements['input'], keyof MaskedInputProps> {}
```

Comment:
Why is `keyof` being used in this refactoring?

Reply:
Please refer to #5085, the reasining is the same.